velox/docs/functions/spark/json.rst

```diff
@@ -0,0 +1,13 @@
+==============
+Json Functions
+==============
+
+.. spark:function:: json_object_keys(jsonString) -> array(string)
+
+    Returns all the keys of the outermost JSON object as an array if a valid JSON object is given. If it is any other valid JSON string, an invalid JSON string or an empty string or is not a JSON object string, the function returns null. ::
+
+        SELECT json_object_keys('{}'); -- []
+        SELECT json_object_keys('{"name": "Alice", "age": 5, "id": "001"}'); -- ['name', 'age', 'id']
```

Comment:
Actually this query lacks '\\' before `"` to work in Spark sql. Could you confirm and do we need to modify?

> scala> spark.sql("SELECT json_object_keys('{"name": "Alice", "age": 5, "id": "001"}');").show(false)
<console>:1: error: ')' expected but string literal found.
       spark.sql("SELECT json_object_keys('{"name": "Alice", "age": 5, "id": "001"}');").show(false)

> spark.sql("SELECT json_object_keys('{\\"name\\": \\"Alice\\", \\"age\\": 5, \\"id\\": \\"001\\"}');").show(false)
+----------------------------------------------------------+
|json_object_keys({"name": "Alice", "age": 5, "id": "001"})|
+----------------------------------------------------------+
|[name, age, id]                                           |
+----------------------------------------------------------+

Reply:
If we use `'`, we do not need '\\' before `"`